_includes/mobile/intro-custom.md

```diff
@@ -0,0 +1,79 @@
+## Overview
+
+You can use the Prebid SDK to monetize your app with a custom ad server or even without an ad server at all. 
+
+This guide outlines how to integrate the Prebid SDK into your app if you don't fit into any of the other approaches documented on the site. Since there are
+many unknown details about your specific situation, guidance will
+remain general.
+
+General components:
+
+- **Prebid SDK** collects parameters and sends to Prebid Server.
+- **Prebid server** handles the bidding and auction process.
+- An ad server or mediation platform without specific Prebid Mobile integration instructions.
+    - If this ad server has an SDK, great, otherwise, your app code will be responsible for crafting requests to the ad server based on response data pulled from Prebid SDK.
+    - It's possible for your app code to just make the ad decision itself, utilizing no ad server at all.
+- **Rendering method** - the first big decision is about how Prebid ads will be rendered:
+    - Either in a webview with the Prebid Universal Creative
+    - Or delegate the rendering to Prebid SDK
+    - The tradeoffs are discussed below
+
+### Render the Winning Ad
+
+We recommend that you study the flows documented in these different rendering approaches:
+
+- [Bidding-Only](/prebid-mobile/pbm-api/{{include.platform}}/{{include.platform}}-sdk-integration-gam-original-api.html#how-it-works) - generally this approach utilizes webviews and the Prebid Universal Creative to render most types of ads.
+- [Prebid-Rendered](/prebid-mobile/modules/rendering/{{include.platform}}-sdk-integration-gam.html) - with this approach the Prebid SDK will render any bids that are chosen for display.
```

Comment:
This approach is irrelevant to the integration with Custom Server. Publishers can't just replace the GMA sd with a custom one. The flow in the "Prebid-Rendered" approach is relevant only to the GAM platform and its features. So, this item and lines 28-31 can mislead integrators.

Reply:
This particular page utilizes the same tiles in a different manner:
- Bidding only 
    - in the beginning of the doc it refers to the GAM SDK + PUC integration scenario 
    - then it refers to a scenario with a custom server that does not necessarily use PUC. The custom server can process the raw bid and render it's `adm`.
- Prebid Rendered
    - in the beginning of the doc it refers to the GAM SDK + Prebid Event Handlers + Prebid SDK integration scenario 
    - then it refers to the Prebid SDK Rendering API.
    
It can confuse a reader a bit, too. 
   
I think that we should skip the detailed description of GAM-based scenarios in this integration approach.

Reply:
@YuriyVelichkoPI - sorry, not following. Honestly I got lazy here, honestly don't understand the scenario, and didn't want to wind up with a lot of duplicate text. And I question whether anyone is actually going to do this.

So my goal is to provide an outline of some things to think about. If you're saying that the "Prebid Rendered" scenario is not supported, fine, I can remove all references to that.

Reply:
If the goal was to provide info about other scenarios on this page - we can keep everything as is.